app/controllers/internal/organizations_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Internal::OrganizationsController < Internal::ApplicationController
+  layout "internal"
+
+  def index
+    @organizations = Organization.order("name DESC")
+
+    return if params[:search].blank?
+
+    @organizations = Organization.where(
```

Comment:
have you considered making this a class method in Organization, so it could be also tested at model level?

Reply:
We will probably redo search in internal at some point, in my opinion, this is just a short term solution for the one or two people who will be hitting this controller.